pythran/pythonic/numpy/random/bytes.hpp

```diff
@@ -0,0 +1,34 @@
+#ifndef PYTHONIC_NUMPY_RANDOM_BYTES_HPP
+#define PYTHONIC_NUMPY_RANDOM_BYTES_HPP
+
+#include "pythonic/include/numpy/random/bytes.hpp"
+#include "pythonic/include/numpy/random/generator.hpp"
+
+#include "pythonic/types/str.hpp"
+#include "pythonic/utils/proxy.hpp"
+
+#include <random>
+
+namespace pythonic
+{
+  namespace numpy
+  {
+    namespace random
+    {
+
+      types::str bytes(long length)
+      {
+        types::str result;
+        result.reserve(length);
+        std::uniform_int_distribution<char> distribution{0, 256};
+        for(long i=0; i<length; i++)
+          result.push_back(distribution(details::generator));
```

Comment:
You're right, let's mesure!
```
    std::string s;
    s.resize(n + 1);
    for (int j = 0; j < n; ++j)
      s[j] = (j);
```
 runs almost three times faster than

```
    std::string s;
    s.reserve(n + 1);
    for (int j = 0; j < n; ++j)
      s.push_back(j);
```

on my laptop.